hip/static/styles/pages/data_report_archive_detail_page.scss

```diff
@@ -1,6 +1,5 @@
 @import "../mixins";
 
-.reports-archive-detail-page-mainarea-hip {
-  @include headers;
-  @include paragraphs;
+.reports-archive-detail-page-hip {
```

Comment:
It looks like `reports-archive-detail-page-hip` is not used in any HTML template, so these styles aren't applied on the report archive detail page

Reply:
FIXED